src/mlmc/quantity_estimate.py

```diff
@@ -1,312 +0,0 @@
-import numpy as np
```

Comment:
This file was replaced by `mlmc.estimator`
It has nothing common with future `quantity_estimate` file.